src/pages/blog/magma-dev-conf-videos.md

```diff
@@ -13,15 +13,14 @@ We had over 580 people join us from 58 countries around the world on February 3r
 [Opening Keynote](https://www.youtube.com/watch?v=SaXoLBXUDHs&list=PLKqaoAnDyfgpCPTI9xDxxx2vuJsB1871N&index=16)
 <br>&nbsp;&nbsp;Yael Maguire, Facebook Connectivity
 
-**Shah, Jonathan, Irfan and Arpit discuss Magma joining Linux Foundation**
+[Shah, Jonathan, Irfan and Arpit discuss Magma joining Linux Foundation](https://www.youtube.com/watch?v=d6qvuCL9608&list=PLKqaoAnDyfgpCPTI9xDxxx2vuJsB1871N&index=19)
 <br>*Video still in production - link will be updated soon*
```

Comment:
I think this line needs to be removed now

Reply:
Thanks Jonathan - fixed now.  New preview: https://deploy-preview-64--magma-website.netlify.app/blog/